README.md

```diff
@@ -1 +1,9 @@
 # hw01
+## Omair Gill's Biography
+![](C:/Users/omair/OneDrive/Desktop/cfss/hw01/Pic.jpg)
+
+I was born and raised in *San Mateo, California*. I attended my undergraduate studies at University of California, Davis and double majored in **Economics and International Relations**. Please see [my Linkedin profile] (https://www.linkedin.com/in/omair-gill-1b0038164) for more information on my professional background. My hobbies include:  
```

Comment:
Hi Omair,

Maybe try having another space after you put the LinkedIn URL? Also it's a good idea to re-examine your work after you submit it!

Anyways, good job!